src/unix/linux_like/linux/mod.rs

```diff
@@ -1,5 +1,8 @@
 //! Linux-specific definitions for linux-like values
 
+use core::cfg;
+use core::mem::size_of;
```

Comment:
`size_of` should be in the prelude. Isn't `cfg` already available?

Reply:
Actually, cfg is not available already, it seems like it has to be explicit otherwise build fails..